src/Slub/Infrastructure/Chat/Slack/ProcessTRRequestListener.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Slub\Infrastructure\Chat\Slack;
+
+use Psr\Log\LoggerInterface;
+use Slub\Application\Common\ChatClient;
+use Slub\Application\PutPRToReview\PutPRToReview;
+use Slub\Application\PutPRToReview\PutPRToReviewHandler;
+use Slub\Domain\Entity\Channel\ChannelIdentifier;
+use Slub\Domain\Entity\PR\PRIdentifier;
+use Slub\Domain\Query\GetPRInfoInterface;
+use Slub\Domain\Query\PRInfo;
+use Slub\Infrastructure\VCS\Github\Query\GithubAPIHelper;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\Routing\RouterInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author    Samir Boulil <samir.boulil@gmail.com>
+ */
+class ProcessTRRequestListener
```

Comment:
Rename ProcessTRSubmissionListener

Reply:
Add E2E tests for:
- no url
- URL correctly formatted but invalid PR Link (can't fetch details)
- Nominal: PR Submitted

Reply:
Consider, Reorganizing infra/chat/slack/* with sub directories (oAuth, TR submission, common, client ?)